Packs/FortiGate/ReleaseNotes/2_0_2.md

```diff
@@ -0,0 +1,141 @@
+
+#### Modeling Rules
+
+##### Fortinet FortiGate Modeling Rule
+
+- Updated XDM mappings for the following fields: 
+| CEF Log Field Name      | Fortigate Field Name | Field Description     | XDM Target Field| 
+| :---        | :----       | :---          | :---       | 
+`FTNTFGTapp`|app|Application Name|`xdm.target.application.name` ~~`xdm.source.application.name`~~|
```

Comment:
Is all this information really needed in the release notes?

Reply:
@ShirleyDenkberg Thanks for the quick feedback. 
I updated the release notes to be more concise. Can you please re-review ?

Reply:
Much better.  Thanks.